scripts/core_tests_list.json

```diff
@@ -91,7 +91,8 @@
     "test_iticket",
     "test_itouch",
     "test_itree",
-    "test_itrim",
+    "test_itrim.Test_Itrim",
+    "test_itrim.test_itrim_magic_with_four_replicas__issue_7468",
```

Comment:
Can "magic" be expanded to better express what the test is about?

Reply:
Sounds good to me.

Reply:
Done